### ISSUE TITLE
Can't get current Linux to install

### ISSUE THREAD

**Issue by kaiakrystal** [flameshot-org/flameshot#3245]
### Flameshot Version

Still getting 11.0.0-2

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

PoPOS 22.04

### Description

Appimage from website is used to install Flameshot. Terminal does not recognize it's installed. Cannot set custom key command.
Use Flatpack to install flameshot. Terminal does not recognize it's installed. Cannot set custom key command
Use terminal to install flameshot. It installs the old version.
Terminal recognizes old version install.

Searching github regarding this has not given me results.

### Steps to reproduce

sudo apt install flameshot
[sudo] password for kaiakrystal: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  grim
The following NEW packages will be installed:
  flameshot grim
0 upgraded, 2 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/567 kB of archives.
After this operation, 2,289 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Selecting previously unselected package flameshot.
(Reading database ... 215178 files and directories currently installed.)
Preparing to unpack .../flameshot_11.0.0-2_amd64.deb ...
Unpacking flameshot (11.0.0-2) ...
Selecting previously unselected package grim.
Preparing to unpack .../grim_1.4.0+ds-1_amd64.deb ...
Unpacking grim (1.4.0+ds-1) ...
Setting up flameshot (11.0.0-2) ...
Setting up grim (1.4.0+ds-1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
Processing triggers for man-db (2.10.2-1) ...
Processing triggers for desktop-file-utils (0.26-1ubuntu3) ...

### System Information

Linux lestra-sys 5.15.0-57-generic #63-Ubuntu SMP Thu Nov 24 13:43:17 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by mmahmoudian** [comment]
Ok, welcome to the Linux world.

So it seems you have wrong understanding of Linux. So let me explain it to you:

You can install a software on Linux in various ways:

1. Your package manager (in case of Ubuntu is `apt`)
2. Flatpak
3. Snap
4. AppImage

Remember that you can use all of them **at the same time** and have 4 or even more versions of Flameshot side-by-side! This is messy and this is what you have done.

## apt

You install with

```sh
sudo apt install flameshot
```

And you run using

```sh
flameshot

flameshot gui
```

## Flatpak

Install:

```sh
flatpak install flathub org.flameshot.Flameshot
```

Run:

```sh
flatpak run org.flameshot.Flameshot

flatpak run org.flameshot.Flameshot gui
```

## Snap

Install:

```sh
snap install flameshot
```

Run:

```sh
flameshot

flameshot gui
```

## AppImage

Installation is just downloading the file. That simple.

Read more here:

https://flameshot.org/docs/installation/installation-linux/#appimage


I will be happy to explain to you basics of Linux, but this is not a bug or issue with Flameshot. So I'm going to close this issue.

**Comment by kaiakrystal** [comment]
No, this does not resolve the issue. You shouldn't close an issue that is
still an issue.

I cannot run your flatpack instructions. "flathub" is not found as a remote
reference.
snap gives me a "snap" command not found
install flameshot just installs the old version.
If I run the appimage to install, I still cannot get a keyboard shortcut to
run it, even with the files in place.

***@***.***:~$ flameshot --version  (this is WITH the appimage
installed)
Command 'flameshot' not found, but can be installed with:
sudo apt install flameshot
***@***.***:~$ ~/.config/flameshot/flameshot.ini
bash: /home/kaiakrystal/.config/flameshot/flameshot.ini: Permission denied
***@***.***:~$ sudo ~/.config/flameshot/flameshot.ini
[sudo] password for kaiakrystal:
sudo: /home/kaiakrystal/.config/flameshot/flameshot.ini: command not found
***@***.***:~$ sudo flatpak install flathub org.flameshot.Flameshot
[sudo] password for kaiakrystal:
Looking for matches…
error: No remote refs found for ‘flathub’
***@***.***:~$ sudo snap install flameshot
sudo: snap: command not found
***@***.***:~$ snap install flameshot
Command 'snap' not found, but can be installed with:
sudo apt install snapd
***@***.***:~$

Does this look resolved to you???

**Comment by mmahmoudian** [comment]
> Does this look resolved to you???

Yes, because you didn't read my comment and you didn't read the link I gave you.

Again, this is you not understanding how Linux works and I'll be happy to help, but you should start reading as well.

When you download the AppImage, you should **run the AppImage file** not `flameshot`. Again, read the link.

About Flatpak, google the error and you will see how to add the flathub:

https://superuser.com/questions/1789203/how-to-solve-error-no-remote-refs-found-similar-to-flathub

The `.ini` file is just a config file, it is not flameshot. It is a normal text file and you can see its content by:

```sh
less ~/.config/flameshot/flameshot.ini
```

Also, never ever ever use `sudo` on flameshot or the softwares you don't trust. Flameshot does not need sudo to work.

**Comment by kaiakrystal** [comment]
So, the initial link to install the flameshot still did not work.
it did not recognize the steps I was trying to make.
So, reading the comment I did, following the instructions on the linked
page, I did. Did it work? No.

The link you sent about Flatpak not recognizing flathub. Done. Flameshot
installed? Done. Restarted pc.
Create a keyboard command to make it run? Nope.
Run flameshot gui in terminal? Nope. Doesn't recognize it's there

Regardless, It wasn't just a matter of not understanding how linux works,
it was the lack of knowledge on what to search for to make it work. Which
you didn't provide in the first response.

...@pop-os:~$ flameshot --version
Command 'flameshot' not found, but can be installed with:
sudo apt install flameshot (which installs the old version)
...@pop-os:~$

All I want is a keyboard command that will work. It doesn't recognize the
command "flameshot gui".

So I followed the instructions to try and figure things out.
https://i.imgur.com/rnRQ8MH.png

https://i.imgur.com/KFrPPlf.png

Still can't find the command to run Flameshot so I can create a keyboard
shortcut.
